07-Modelling-Patchy-Particles/02-Simulating-a-System-of-Patchy-Particles.ipynb

```diff
@@ -0,0 +1,1909 @@
+{
```

Comment:
<p>I don't actually see a figure showing up here. All I see is something that says "&lt;Figure size 360x222.48 with 2 Axes&gt;". Is it going to show up in the tutorial this way, or will we see the whole figure?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/glotzerlab/hoomd-examples/pull/84/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='41'/>

Reply:
See the figure here: https://github.com/glotzerlab/hoomd-examples/blob/patchy-particle-tutorial/07-Modelling-Patchy-Particles/02-Simulating-a-System-of-Patchy-Particles.ipynb
ReviewNB doesn't display svg images.